src/docs/test/assign.test.js

```diff
@@ -0,0 +1,338 @@
+const app = require("../../../app.js");
+const mocha = require("mocha");
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const jwt = require("jsonwebtoken");
+const dotenv = require('dotenv');
+dotenv.config({path:'./config.env'})
+
+chai.should();
+chai.use(chaiHttp);
+const { expect } = chai;
+const { it, describe } = mocha;
+
+// import fs from 'fs'
+// import mongoose from 'mongoose'assign/2290cccd-d5ef-4be8-81d3-0c63d9df4c22/54535156-f6ee-4454-9b58-4f33d9f79cab;
+
+const signToken = (uuid) => {
+    return jwt.sign({ uuid }, process.env.JWT_SECRETE, {
+      expiresIn: process.env.JWT_EXPIRES_IN,
+    });
+  };
+const token = signToken('9e3d2a6c-5484-4d86-8eb0-f0098cd7a540');
+console.log(token)
+
+
+before(function(done){
+    this.timeout(20000);
+    chai.request(app)
+            .put('/api/v1/assignment/assign/9e3d2a6c-5484-4d86-8eb0-a0098cd7a540/2ae55c54-1a1a-4eed-8f8e-a6c1a62e6d08')
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: `Bearer ${token}` })
+           
+            .end((err, res) => {
+                if(err){
+                    console.log(err)
+                } 
+                // done(err) 
+                // console.log(res)                   
+                chai.expect(res).have.status(200);
+                chai.expect(res.body).be.a('object');
+                chai.expect(res.body).to.have.deep.property("message")
```

Comment:
is this a test setup (before hook?) or a test case? we do not need to compare expected and actual test results in hooks, only in test cases